packages/lit-html/src/directive-helpers.ts

```diff
@@ -59,6 +64,16 @@ export const isDirectiveResult = (
     ? (value as DirectiveResult)?._$litDirective$ !== undefined
     : (value as DirectiveResult)?._$litDirective$ === klass;
 
+/**
+ * Tests is a part is a single-expression part.
+ *
+ * Only AttributePart and PropertyPart can by multi-valued, when the expression
+ * that created them is an interpolation. Multi-expression parts have a
```

Comment:
The phrasing "when the expression that created them is an interpolation" reads strange to me. I'm not sure defining "multi-valued" is helping here, because being multi-valued (if that means `values.length > 1`) isn't the only way for this to return false. It could also be `values.length===1` but the strings were also empty.

Reply:
removed that clause